analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -163,6 +163,23 @@ open class Npm(
         }
     }
 
+    private fun parseLicenses(json: JsonNode): SortedSet<String> {
+        val declaredLicenses = sortedSetOf<String>()
+
+        // See https://docs.npmjs.com/files/package.json#license.
+        json["license"]?.let { licenseNode ->
+            val type = licenseNode.textValue() ?: licenseNode["type"].textValueOrEmpty()
+            declaredLicenses += type
+        }
+
+        // Some old packages used license objects or a "licenses" property containing an array of license objects.
```

Comment:
Doesn't the "license objects" part of this comment belong to the code above?

Reply:
Done.